### ISSUE TITLE
NullReferenceException occurs when mapping air starsheets with the crafted converter

### ISSUE THREAD

**Issue by mbnkwm** [Beamographic/rush#281]
__osu! version:__ 2022.1123.0
__Rush! version:__ 2022.1118.0

# Steps

1. Create an osu! beatmap with `crafted` in tags
2. Place a slider in either the air or dual section (top left / middle left)
3. Save and return to song select

__Expected:__ A beatmap is made with air/dual starsheets 

__Actual:__ The following error occurs and the map isn't playable until the sliders are removed from the air/dual section

# Error

```
2022-12-03 14:14:22 [error]: Failed to calculate beatmap difficulty
2022-12-03 14:14:22 [error]: System.NullReferenceException: Object reference not set to an instance of an object.
2022-12-03 14:14:22 [error]: at osu.Game.Rulesets.Rush.Objects.StarSheet.updateNestedSamples()
2022-12-03 14:14:22 [error]: at osu.Game.Rulesets.Rush.Objects.StarSheet.CreateNestedHitObjects(CancellationToken cancellationToken)
2022-12-03 14:14:22 [error]: at osu.Game.Rulesets.Objects.HitObject.ApplyDefaults(ControlPointInfo controlPointInfo, IBeatmapDifficultyInfo difficulty, CancellationToken cancellationToken) in /var/lib/buildkite-agent/builds/debian-gnu-linux-vm-1/ppy/osu/osu.Game/Rulesets/Objects/HitObject.cs:line 129
2022-12-03 14:14:22 [error]: at osu.Game.Beatmaps.WorkingBeatmap.GetPlayableBeatmap(IRulesetInfo ruleset, IReadOnlyList`1 mods, CancellationToken token) in /var/lib/buildkite-agent/builds/debian-gnu-linux-vm-1/ppy/osu/osu.Game/Beatmaps/WorkingBeatmap.cs:line 299
2022-12-03 14:14:22 [error]: at osu.Game.Rulesets.Difficulty.DifficultyCalculator.preProcess(IEnumerable`1 mods, CancellationToken cancellationToken) in /var/lib/buildkite-agent/builds/debian-gnu-linux-vm-1/ppy/osu/osu.Game/Rulesets/Difficulty/DifficultyCalculator.cs:line 171
2022-12-03 14:14:22 [error]: at osu.Game.Rulesets.Difficulty.DifficultyCalculator.Calculate(IEnumerable`1 mods, CancellationToken cancellationToken) in /var/lib/buildkite-agent/builds/debian-gnu-linux-vm-1/ppy/osu/osu.Game/Rulesets/Difficulty/DifficultyCalculator.cs:line 65
2022-12-03 14:14:22 [error]: at osu.Game.Beatmaps.BeatmapDifficultyCache.computeDifficulty(DifficultyCacheLookup& key, CancellationToken cancellationToken) in /var/lib/buildkite-agent/builds/debian-gnu-linux-vm-1/ppy/osu/osu.Game/Beatmaps/BeatmapDifficultyCache.cs:line 263
```

Seems to be because `NodeSamples` is never assigned for [air starsheets](https://github.com/Beamographic/rush/blob/ad6be678444d9081a760d62ed653a3882c3eba87/osu.Game.Rulesets.Rush/Beatmaps/RushCraftedBeatmapConverter.cs#L104-L110), only [ground starsheets](https://github.com/Beamographic/rush/blob/ad6be678444d9081a760d62ed653a3882c3eba87/osu.Game.Rulesets.Rush/Beatmaps/RushCraftedBeatmapConverter.cs#L119)

**Comment by LumpBloom7** [comment]
Having a quick look into it, it seems to be the case. Will fix in upcoming release.

If it is not too much, could I have a copy of the beatmap that causes this error, for testing?

**Comment by LumpBloom7** [comment]
works for me, no worries

**Comment by LumpBloom7** [comment]
Fixed in #282

**Comment by LumpBloom7** [comment]
@mbnkwm The fix is now available in the latest rush release, do give it a whirl.
